fml/trace_event.cc

```diff
@@ -65,8 +65,8 @@ void TraceSetTimelineMicrosSource(TimelineMicrosSource source) {
 }
 
 size_t TraceNonce() {
-  static std::atomic_size_t gLastItem;
-  return ++gLastItem;
+  static std::atomic_size_t last_item;
+  return ++last_item;
```

Comment:
We don't seem to be consistent with scoped statics -- i.e. whether we name them the same as globals within global scope and add a `g` prefix, or name them the same as locals. There are semantic differences to both. Personally I think it's pretty clear that it's static, at least in this case. My guess is that _most_ cases of function-local statics are similarly concise.

clang-tidy does support specifying static local/field formatting/prefixes if we want to give these special treatment. Opinions?

Reply:
FWIW for `static const` we tend to prefix as `kFooBar`.

For static fields it looks like typically we don't prefix:
* common/graphics/persistent_cache.h:145:  static std::atomic<bool> cache_sksl_;
* common/graphics/persistent_cache.h:149:  static std::atomic<bool> strategy_set_;
* display_list/display_list_complexity_metal.h:95:  static DisplayListMetalComplexityCalculator* instance_;

For static locals I see a mix:
* display_list/display_list_unittests.cc:684:    static DisplayListBuilder builder;
* flow/flow_test_utils.cc:9:static std::string gGoldenDir;
* flow/flow_test_utils.cc:10:static std::string gFontFile;
*